pkg/awsutils/vpc_ip_resource_limit.go

```diff
@@ -148,6 +148,22 @@ var InstanceENIsAvailable = map[string]int{
 	"m5d.16xlarge":  15,
 	"m5d.24xlarge":  15,
 	"m5d.metal":     15,
+	"m5dn.large":    3,
```

Comment:
We need the number of IPs per ENI for all types as well in this map:
```
var InstanceIPsAvailable = map[string]int{
```

Reply:
Ah, I forget to add this part. Just add to the list

Reply:
Thanks!